pkg/v1/cli/command/core/repo.go

```diff
@@ -23,7 +23,7 @@ var repoCmd = &cobra.Command{
 	Short: "Manage plugin repositories",
 }
 
-func init() {
+func init() { //nolint:dupl
```

Comment:
Add explaining comment, e.g. ` // the package has multiple files with an init() function`. Perhaps even on the same line.